readme.md

```diff
@@ -51,7 +51,7 @@ This will:
 
 You should be able to launch the app on the platform you built for. If not, check your settings and try again.
 
-**Be careful** not to include node_modules you don't want into your final app. `Electron-packager`, `electron-prebuilt` and `.git` will be ignored by default. You can use `--ignore` to ignore files and folders, e.g. `--ignore=node_modules/electron-packager`.
+**Be careful** not to include node_modules you don't want into your final app. `Electron-packager`, `electron-prebuilt` and `.git` will be ignored by default. You can use `--ignore` to ignore files and folders, e.g. `--ignore=node_modules/electron-packager` or `--ignore="(node_modules/atom-shell|node_modules/node-gyp)"`
```

Comment:
`atom-shell` is no more, so find a better example module.

And can be simplified to:

`--ignore="node_modules/(atom-shell|node-gyp)"`